src/Compilers/CSharp/Portable/CodeGen/EmitArrayInitializer.cs

```diff
@@ -469,5 +476,43 @@ private int TryGetRawDataForArrayInit(BoundArrayInitialization initializer, out
             data = writer.ToImmutableArray();
             return elementCount;
         }
+
+        /// <summary>
+        ///  Returns a byte blob that matches serialized content of zero-valued array.
+        /// </summary>
+        private int TryGetRawDataForDefaultArray(BoundArrayCreation initializer, out ImmutableArray<byte> data)
```

Comment:
>TryGetRawDataForDefaultArray [](start = 20, length = 28)

Can we share common logic with `TryGetRawDataForArrayInit`?

Reply:
Done.